apps/src/templates/sectionProgress/progressTables/progressTableHelpers.js

```diff
@@ -111,33 +138,33 @@ function formatTimeSpent(studentProgress) {
     const minutes = studentProgress.timeSpent / 60;
     return `${Math.ceil(minutes)}`;
   }
-  return missingDataFormatter(studentProgress, 'timeSpent');
+  return missingDataFormatter(!!studentProgress);
 }
 
 function formatLastUpdated(studentProgress) {
   if (studentProgress?.lastTimestamp) {
     return moment.unix(studentProgress.lastTimestamp).format('M/D');
   }
-  return missingDataFormatter(studentProgress, 'lastTimestamp');
+  return missingDataFormatter(!!studentProgress);
 }
 
 /**
- * Handle formatting for each of three distinct cases:
- * 1) `studentProgress` is null: this means the student hasn't started the
- *    level/lesson, so we display nothing.
- * 2) `studentProgress[field] === 0`: this is a special case that only
- *    applies to lesson progress. it means the student has started the lesson,
- *    but we don't have data in this field, in which case we display nothing.
- * 3) `studentProgress[field]` is null: this case only applies to level
- *    progress, and means we have progress data but we don't track `field` for
- *    this level type, which we indicate by displaying '-'.
+ * If the student hasn't made any progress on the level, we display nothing.
+ * However, if the student has made progress but we don't have time spent or
+ * last update data, that means we aren't tracking that data for this level so
+ * we display '-' to indicate N/A.
+ *
+ * Note: we also use '-' for lessons with no levels.
  */
-function missingDataFormatter(studentProgress, field) {
-  if (!studentProgress || studentProgress[field] === 0) {
-    return '';
-  } else if (!studentProgress[field]) {
+function missingDataFormatter(progressNotApplicable) {
+  if (progressNotApplicable) {
     return '-';
```

Comment:
Could return `emptyLessonFormatter()` here

Reply:
`emptyLessonFormatter()` returns an em dash, whereas here we return a hyphen. i can add comments since that's a little hard to notice visually.